cmd/cloud/cluster.go

```diff
@@ -264,3 +279,46 @@ var clusterShowStateReport = &cobra.Command{
 		return nil
 	},
 }
+
+func getRequestedUtilityMetadata(command *cobra.Command) map[string]string {
```

Comment:
This should be be handled server side. i.e. we should do any default-setting when the ProvisionClusterRequest is constructed from the reader. This allows non-cloud-cli behavior to be more similar to simply sending raw JSON payloads and also allows us to do everything in one place.

Look at the CreateClusterRequest from reader for an example on default and validation.
https://github.com/mattermost/mattermost-cloud/blob/master/model/cluster_request.go#L23

Reply:
Moved the part that has to do with defaults and isn't just basic processing to the api side